modules/scanning-images.adoc

```diff
@@ -172,6 +172,29 @@ Scanner can check for vulnerabilities in dependencies for the following programm
 |===
 . Only supported in Scanner V4.
 
+[id="supported-layer-compression-formats_{context}"]
+== Supported layer compression formats
+
+Container image layers are compressed. StackRox Scanner and Scanner V4 support different formats as shown in the following table:
```

Comment:
I was a bit confused by `Container image layers are compressed.` followed by the table containing `No compression`. Would something like this be clearer?
```suggestion
Container image layers are `tar` archives that may or may not be compressed. StackRox Scanner and Scanner V4 support different formats as shown in the following table:
```